src/io/github/sspanak/tt9/languages/definitions/Yiddish.java

```diff
@@ -0,0 +1,16 @@
+package io.github.sspanak.tt9.languages.definitions;
+
+import java.util.Locale;
+
+public class Yiddish extends Hebrew {
+	public Yiddish() {
+		super();
+
+		locale = new Locale("ji","JI");
+		dictionaryFile = "ji-utf8.csv";
+		hasUpperCase = false;
+		isPunctuationPartOfWords = true;
+
+
```

Comment:
```suggestion
```
```suggestion
```

These two are already set in the parent constructor, so it shouldn't be necessary to assign them again.

Reply:
I originally didn't have those in due to your reasoning but when I tried compiling in Android Studio it gave me a library error due to the punctuation so I just added it to make that error go away. If it really doesnt need it then I will take it out.